service/src/routes/api.ts

```diff
@@ -0,0 +1,62 @@
+import express from 'express'
+import dotenv from 'dotenv'
+import axios, { AxiosInstance, AxiosResponse }from 'axios'
+
+export const register = (app: express.Application) => {
+
+    //Get a list of tasks from Mavenlink
+    app.get("/tasks", async (req, res) => {
+        const options = {
+            headers:{
+                Authorization: "Bearer " + process.env.MAVENLINK_TOKEN
+            }
+        };
+        axios.get(process.env.MAVENLINK_TASK_URL, options)
+        .then(response => {
+            console.log(response.data.stories);
```

Comment:
All requests are left hanging because a response is never sent.

We could do the following if the axios call is successful:
``` suggestion
            res.status(200).send(response.data.stories);
```